src/Relude/List/Reexport.hs

```diff
@@ -7,7 +7,7 @@ Copyright: (c) 2016 Stephen Diehl
 License:    MIT
 Maintainer: Kowainik <xrom.xkov@gmail.com>
 
-This module reexports functinons to work with list and 'NonEmpty'.
+Reexports most of the "Data.List" and "Data.List.NonEmpty".
```

Comment:
Are these "" are intentional?

Reply:
@vrom911 yes, they give links to modules. In Haddock we need to use single quotes to link to function or types and double quotes to link to modules.